packages/suite/src/components/suite/modals/ImportTransaction/index.tsx

```diff
@@ -7,6 +8,10 @@ import { DelimiterForm } from './components/DelimiterForm';
 import { ExampleCSV } from './components/ExampleCSV';
 import { parseCSV } from '@suite-common/wallet-utils';
 
+const StyledModal = styled(Modal)`
+    width: 600px;
+`;
+
 type Props = {
```

Comment:
![Screenshot 2022-09-05 at 10 15 09](https://user-images.githubusercontent.com/33235762/188401656-0b7c1035-c8af-4cdd-90a4-fea6a386095d.png)
Could you please fix this for a phone?

Reply:
22598792797b8c4022b75724baf2ef5f6bef4842